pylib/anki/notes.py

```diff
@@ -75,6 +76,14 @@ def __repr__(self) -> str:
     def joinedFields(self) -> str:
         return joinFields(self.fields)
 
+    def card(self, card: int, reload: bool = False) -> Optional[anki.cards.Card]:
+        if reload or card not in self._cards:
+            try:
+                self._cards[card] = self.col.getCard(self.card_ids()[card])
+            except IndexError:
+                return None
+        return self._cards[card]
+
```

Comment:
I fear this may lead to accidental cache coherency issues down the road. If the browser needs to cache the lookup, it might be better to explicitly do it at the call site, instead of implicitly in a method here. But perhaps we would not even need this if the .data() was calculated by the backend?